app/views/Board.coffee

```diff
@@ -0,0 +1,73 @@
+Cell = require './Cell'
+
+class Board
+
+  constructor: (@grid, @two) ->
+
+    @size = @two.height * .80
+    offset = @size * .025
+    width = ( @size - offset ) / ( @grid.dimension ) - offset
+
+    @x = ( @two.width ) / 2
+    @y = ( @two.height ) / 2
+    @y = if @y < @size / 2 then @size / 2 else @y
+
+    board = @two.makeRectangle @x, @y, @size, @size
+    board.noStroke().fill =  '#c2d1e6' #'#B0C4DE' #'#F0F8FF' #'#CDE1FF'
```

Comment:
We should probably have global colors so we can reference them instead of trying to interpret the hex values